### DISCUSSION TITLE
共有広場

### DISCUSSION THREAD

**Discussion by Ayato-kosaka** [Ayato-kosaka/react-itinerary#543] [General]
共有すべきことをメモする

**Comment by Ayato-kosaka** [comment]
@Takapy1 
GitHubのアプリにGitHub project が追加されたみたい！

![image](https://user-images.githubusercontent.com/53715839/225221637-1bcfa092-f538-4f09-800d-3bb99503bc84.png)

**Comment by Ayato-kosaka** [comment]
@Takapy1 
**勤怠連絡**

6/7-6/12で旅行に行くから不在です。
ミーティングの日程は日にちが近づいてきたら調整させて〜

**Comment by Ayato-kosaka** [comment]
@Takapy1 
申し訳ない！GitHub Project ぶっ壊れて元に戻すから、今日は変更しないでほしい！

**Reply by Takapy1** [reply]
@Ayato-kosaka 
あ、だよね
俺も使ってて死んでると思った
よかった

**Reply by Ayato-kosaka** [reply]
@Takapy1 
申し訳ない！修復終えたから、編集しても大丈夫！
使い方明日また説明するけど、↓で編集しても反映されるようにしておいた！
https://docs.google.com/spreadsheets/d/1K6CPWRJOPC2AhoIGD44IfIuUqzyZiZgMNGvg4JAPTWE/edit#gid=360033118
